### ISSUE TITLE
Chrome reports touch support when it is not supported

### ISSUE THREAD

**Issue by doctor-g** [playn/playn#27]
Running PlayN HTML builds on Chrome, on a non-touch laptop, PlayN is reporting that the platform supports touch. As a result (and related to https://github.com/threerings/tripleplay/issues/58), TriplePlay buttons are not clickable on mouse-driven, non-touch environments.

This behavior is not seen on Firefox, which correctly reports no touch support, which makes me wonder if the root problem is how touch support is inferred around here: https://github.com/playn/playn/blob/67cba67fe48c7c42c6494085d76b85a40ad99520/html/src/playn/html/HtmlInput.java#L183

**Comment by doctor-g** [comment]
TL;DR: Not your problem.

I spent some time this morning isolating what exactly caused the problem. We've seen it in two scenarios: in a computer lab environment with two-head displays, and in a development environment, with a laptop doubled-up with an external monitor for two-head display. Turns out that on that same laptop, everything worked fine when the second display was not connected. This is a clue.

Turns out the root of the problem has to do with mixing touch- and non-touch monitors. In truth, I didn't even know this monitor hooked up to my (non-touch) laptop was a touch monitor. The lab we were in for testing a project last weekend, it was also set up with heterogeneous monitors---some touch, some not. In Chrome, if one of the two monitors is touch, then the browser reports itself as touch-capable, even if it's on the non-touch display. This has the effect that mouse touches are ignored in PlayN. Firefox, on the other hand, reports itself as non-touch in a heterogeneous environment, which means the mouse clicks still work (and so do touch, in fact.).

The moral of the story is not to mix touch and non-touch monitors. I don't think it's reasonable for PlayN to try to work around this when the browsers themselves are not responding to which display they are connected to in two-display configurations.

**Comment by samskivert** [comment]
Thanks for figuring that out. Computers are complicated! :)
